server/controllers/area.controllers.js

```diff
@@ -23,7 +23,7 @@ const AreaController = {
 
   getArea(req, res, next) {
     const { state } = req.params;
-    getStateAndArea(state)
+    getArea(state)
```

Comment:
Just to point out that we are retrieving multiple areas: The same applies to `getState` too 🙏 

```suggestion
    getAreas(state)
```